### PR TITLE
save table stuff in store + dynamically add stuff from user

### PR DESCRIPTION
**Pull request by kialj876**
Signed-off-by: Kial Jinnah <kialj876@gmail.com>

*Issue #:* /bcgov/entity#11114

*Description of changes:*
- moved needed table items from dashboard into the store
- updated dashboard load flow for keeping existing registrations / adding new dynamically

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/store/mutations/mutations-model.ts [hunk 1]
```diff
@@ -231,5 +235,46 @@ export const mutateShowStepErrors = (state: StateIF, show: boolean) => {
 }
 
 export const mutateRegistrationTable = (state: StateIF, regTableData: RegTableDataI) => {
+  console.log('setting reg table data: ', regTableData)
```
Comment (jordiwes) [inline ppr-ui/src/store/mutations/mutations-model.ts]: remove console?

Comment (kialj876) [reply ppr-ui/src/store/mutations/mutations-model.ts]: done

**Review by jordiwes — APPROVED** [review]
Looks good to me. Big change!

**Review by doug-lovett — APPROVED** [review]
Very nice again.

**Pull request merged**